spec/menu_spec.rb

```diff
@@ -0,0 +1,3 @@
+require 'menu'
+
+# ?? What do I test ?
```

Comment:
I guess a good thing to test here would be the loading of a menu array from a CSV file, e.g:

```ruby
menu = Menu.new("test_menu.csv")
expect(menu).to eq(["Burger", "Pasta", "Salad"])
```